core/new-gui/src/app/workspace/component/navigation/navigation.component.html

```diff
@@ -13,6 +13,8 @@
                placeholder="Untitled Workflow"/>
       </label>
       <span class="save-state" spellcheck="false">{{autoSaveState}}</span>
+        <texera-user-icon class="user-icon-navigation-page">
```

Comment:
1. please move this `texera-user-icon` component outside of `texera-navigation-workflow-metadata`. They should be parallel to each other.
2. add `*ngIf="userSystemEnabled"`. login should not show up if user system is disabled.
3. fix indentations.